packages/superset-ui-connection/src/callApi/callApi.ts

```diff
@@ -1,7 +1,12 @@
 import 'whatwg-fetch';
+import fetchRetry from 'fetch-retry';
 import { CallApi } from '../types';
 import { CACHE_AVAILABLE, CACHE_KEY, HTTP_STATUS_NOT_MODIFIED, HTTP_STATUS_OK } from '../constants';
 
+const RETRY_COUNT = 3;
+const RETRY_DELAY_MS = 1000;
+const RETRY_ON_HTTP_STATUS_CODES = [503];
```

Comment:
do you think we should make this configurable?  like make this the default but override-able? 

we could do that for all of the constants actually. no reason not to expose them for more granular customization?

Reply:
That's a good call, I'll move it into the SupersetClient class.